src/shared/analytics/analytics.ts

```diff
@@ -9,6 +9,7 @@ type MetabaseEvent =
   | 'signed_message'
   | 'signed_transaction'
   | 'client_error'
+  | 'error_registering_dna_action'
```

Comment:
I don't think we need a special event for this

Reply:
ah, ok, let it be just `client_error`